src/components/Modal/Modal-CloseTrigger.tsx

```diff
@@ -0,0 +1,29 @@
+import { purple } from '../../styles/color';
+import Button from '../Button/Button';
+import React, { PropsWithChildren } from 'react';
+import { useModalContext } from '../../hooks/useModalContext.hook';
+
+export interface ModalButtonProps extends PropsWithChildren {
+  customAction?: () => void;
+  buttonColor?: string;
+  minSize?: { width?: number; height?: number };
+}
+
+export const ModalCloseTrigger = ({ children, ...props }: ModalButtonProps) => {
+  const context = useModalContext();
+
+  const handleCloseModal = () => {
+    context.setIsOpen(false);
+  };
+
+  return (
+    <Button
+      children={children ?? '취소'}
+      variant="outline"
+      size="small"
+      color={props.buttonColor ?? purple['500']}
+      minSize={props.minSize ?? { width: 60, height: 40 }}
+      onClick={handleCloseModal}
+    />
```

Comment:
```suggestion
    <Button
      variant="outline"
      size="small"
      color={props.buttonColor ?? purple['500']}
      minSize={props.minSize ?? { width: 60, height: 40 }}
      onClick={handleCloseModal}
    >
      {children ?? '취소'}
    </Button>
```

P3:
작성해주신 방법도 가능하지만,
HTML 태그를 사용하는 것처럼 활용하는것이 가장 이해하기 쉬운 방법 중 하나 아닐까요?